docs/product/components/inputs.html

```diff
@@ -71,6 +71,70 @@
         <li>If you have a group of related inputs, use the <code class="stacks-code">fieldset</code> and <code class="stacks-code">legend</code> to group them together.</li>
     </ul>
     <p class="stacks-copy">For more information, please read Gov.UK's article, <a href="https://accessibility.blog.gov.uk/2016/07/22/using-the-fieldset-and-legend-elements/" target="_blank"><em>"Using the fieldset and legend elements"</em></a>.</p>
+
+    {% header "h3", "Required input fields" %}
+    <p class="stacks-copy">Labels or instructions must be provided when content requires user input. For any input field within a form that is required for successful data submission, provide the asterisk <code class="stacks-code">*</code> as a symbol and a legend advising the meaning of the symbol before the first use.</p>
+    <p class="stacks-copy">In order to ensure the symbol (asterisk) is clearly visible:</p>
+    <ul class="stacks-list">
+        <li>Enlarge the asterisk by <code class="stacks-code">125%</code> compared to the label text</li>
+        <li>Use <code class="stacks-code">red-400</code>
+            {%
+                render 'color-swatch.html',
+                color: "red-400",
+                size: "16",
+                classes: "mbn2"
+            %}
+            for the asterisk symbol</li>
+    </ul>
```

Comment:
When I first read this section, I felt like this was suggesting I apply a font-size and color to the asterisk _instead of_ apply a pre-built solution like the `.s-required-symbol` included in this PR.

> Shall we have a section where we document the new class a bit more formally like we do for validations classes?

Yes, I think that's a good call. I'd expect that to reduce confusion like I experienced. I'd also recommend mentioning the `.s-required-symbol` here as well so we encourage it's use. Maybe something like this added after we mention the styles:
> Stacks includes a `.s-required-symbol` component which includes the above styles.

What do you think @giamir?

Reply:
I copied over from @CGuindon's [draft](https://docs.google.com/document/d/1AHzVBvjiRgPY9p8NAq78HQUitwQr_GlLZ-kSy2IR_mc/edit#bookmark=id.3jj27yq0o9py) without doing a thorough review of the copy. Will make sure to address your point and add the class detailed description. 

Thank you

Reply:
I have adjusted the copy a bit for the inputs docs. Let me know if now it sounds better. 
https://deploy-preview-1723--stacks.netlify.app/product/components/inputs/#required-input-fields

Reply:
LGTM! Thanks for the copy edits and review!